content/transparency-tracker.md

```diff
@@ -0,0 +1,57 @@
++++
```

Comment:
Is this file just left over and the real file is content/topics/transparency-tracker/_index.md?

Reply:
This page was requested to be different from the transparency-tracker topics page:

https://deploy-preview-356--poor-richard-spotlightpa.netlify.app/statecollege/transparency-tracker
https://deploy-preview-356--poor-richard-spotlightpa.netlify.app/topics/transparency-tracker/

Reply:
Got it. Then modify the topics/transparency-tracker/_index.md file to remove the block stuff, since it's not active anymore. Maybe just do

`git checkout master -- content/topics/transparency-tracker/_index.md`

to revert it?